### ISSUE TITLE
Can this replace avahie completly?

### ISSUE THREAD

**Issue by MattL0** [philippe44/mdnssvc#3]
Hi , 

my case is that i have 3 raspi 3b +. They use avahi as a client i think to brodcast the hosname to whoever want it... 

- I also have HomeSeer server that use a plugin. This plugin take adventage of the shairportsync install and it sends them tts files, and they are coordinated. Avahi is also installed on the HomeSeer Server to discover those shairportsync players. 

I get weird errors from avahi on my homeseer instalation, and this prevent the homeseer plugin to turn off properly, when i shutdown my HomeSeer sever. 

---- I am not sure what is the client or server here.. but you like to know if i can replace avahie altogether with TinySVCmDNS ?

**Comment by philippe44** [comment]
Yes and no. Avahi is very well supported, including security fixes. I know that there base I used have somme issues https://bitbucket.org/geekman/tinysvcmdns/issues?status=new&status=open so I would recommend trying to find the problem you have with avahi. TinySVCmDNS make sense for a very targeted use
